dashboard/views.py

```diff
@@ -150,6 +151,10 @@ def file_access_within_week(request):
     logger.info("start time=" + startTimeString + " end_time=" + endTimeString);
     df = pd.read_sql(sqlString, conn, params={"start_time": startTimeString,"end_time": endTimeString})
 
+    # return if there is no data during this interval
+    if (df.empty):
+        return HttpResponse("no data");
```

Comment:
I don't know if this is being grabbed and displayed as part of the UI, if it is then having the response in the JSON format would make it consistent